web_src/js/features/common-global.js

```diff
@@ -44,9 +44,17 @@ export function initFootLanguageMenu() {
 
 
 export function initGlobalEnterQuickSubmit() {
-  $('.js-quick-submit').on('keydown', function (e) {
+  $(document).on('keydown', '.js-quick-submit', function (e) {
     if (((e.ctrlKey && !e.altKey) || e.metaKey) && (e.keyCode === 13 || e.keyCode === 10)) {
```

Comment:
I know it is simply legacy code, however, `keyCode` is deprecated and should be replaced with `key`.
Should this be a part of this PR, or should I open a separate PR? (There are 10 applicable lines in the whole repo...)

Reply:
Probably best to do it here I think.

Reply:
Agree, I am thinking about some more clear code, will make some changes later.